### PR TITLE
added curl healthcheck to dockerfile; changed definition of healthy

### PR DESCRIPTION
**Pull request by bitofant**
added curl-based health check definition to dockerfile; changed defintion of healthy: the container remains healthy even when engine is unavailable

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,6 +13,10 @@ RUN wget ${ARACHNI_DISTRIBUTION} -P /sectools --output-document arachni.tar.gz &
     mv arachni-${ARACHNI_LONG_VERSION} arachni && \
     rm arachni.tar.gz
 
+# healthchecks: requires curl (if not present in container)
+# RUN apk --update --no-cache add curl
```
Comment (MartinLang1) [inline Dockerfile]: (see health check pull request from other scanners)

File: Dockerfile [hunk 2]
```diff
@@ -13,6 +13,10 @@ RUN wget ${ARACHNI_DISTRIBUTION} -P /sectools --output-document arachni.tar.gz &
...
+HEALTHCHECK --interval=3s --timeout=3s --start-period=5s --retries=3 CMD curl --fail http://localhost:8080/status || exit 1
```
Comment (MartinLang1) [inline Dockerfile]: (see health check pull request from other scanners)

File: Dockerfile (hunk 1 above)
Comment (bitofant) [reply Dockerfile]: comments removed in today's commit

File: Dockerfile (hunk 2 above)
Comment (bitofant) [reply Dockerfile]: since today's commit all scanners have the same timings

**Pull request merged**